library/alloc/src/fmt.rs

```diff
@@ -570,6 +570,7 @@ use crate::string;
 ///
 /// [`format_args!`]: core::format_args
 /// [`format!`]: crate::format
+// #[inline]
```

Comment:
Did you want to add this before I start the perf run?

Reply:
No, I tested locally, `inline` will make the code big and bloat. I'm just lazy to remove and rebuild compiler locally.